### PR TITLE
feat(features): copy non attached boundary events as well

### LINKED ISSUE #1154: Allow Copying Boundary Events
Since detaching of boundary events is now possible it should be possible to copy boundary events. This is only prevented by the [`element.copy` rule](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/rules/BpmnRules.js#L902).

When disabling this rule:

foobar

### PR DESCRIPTION
**Pull request by oguzeroglu**
Closes #1154
Requires https://github.com/bpmn-io/diagram-js/pull/410

Kapture 2019-09-20 at 10 15 49

### CODE REVIEW

**Comment by oguzeroglu** [comment]
@philippfromme Ready for review!

**Comment by philippfromme** [comment]
I think adding a test case for copying boundary events makes sense here.

**Comment by oguzeroglu** [comment]
@philippfromme I converted 2 test cases into the new behaviour and they check if the boundary events are copiable now. What other cases do we wanna cover here?

**Comment by oguzeroglu** [comment]
@pinussilvestrus Nice suggestion! Updated the commit message.

**Comment by oguzeroglu** [comment]
@philippfromme Added another test case which only copies a boundary event without a host. Is that what you meant?

File: test/spec/features/copy-paste/BpmnCopyPasteSpec.js [hunk 1]
```diff
@@ -71,6 +71,17 @@ describe('features/copy-paste', function() {
       }));
 
 
+      it('should copy boundary events without host', inject(function() {
```
Comment (philippfromme) [inline test/spec/features/copy-paste/BpmnCopyPasteSpec.js]: I'd prefer a full integration test here. Copy, paste & verify the result.

**Comment by oguzeroglu** [comment]
@philippfromme ready!

File: test/spec/features/copy-paste/BpmnCopyPasteSpec.js [hunk 2]
```diff
@@ -71,6 +71,34 @@ describe('features/copy-paste', function() {
       }));
 
 
+      it('should copy boundary events without host', inject(function(elementRegistry, canvas, copyPaste) {
```
Comment (philippfromme) [inline test/spec/features/copy-paste/BpmnCopyPasteSpec.js]: Last request: we should have test cases for copying boundary events, pasting them onto root AND paste as attacher.

File: test/spec/features/copy-paste/BpmnCopyPasteSpec.js (hunk 1 above)
Comment (oguzeroglu) [reply test/spec/features/copy-paste/BpmnCopyPasteSpec.js]: Ah yes, good idea, I'm on it!

**Comment by oguzeroglu** [comment]
@philippfromme Ready!

File: test/spec/features/copy-paste/BpmnCopyPasteSpec.js [hunk 3]
```diff
@@ -71,6 +71,75 @@ describe('features/copy-paste', function() {
...
+              y: 1000
+            }
+          });
+
+          // then
+          expect(rootElement.children).to.have.length(2);
+
+          expect(copiedElements).to.have.length(1);
+
+          expect(copiedElements[0].type).to.equal('bpmn:BoundaryEvent');
```
Comment (philippfromme) [inline test/spec/features/copy-paste/BpmnCopyPasteSpec.js]: This cannot be a boundary event, it must be an intermediate event.

**Comment by barmac** [comment]
Before we merge this, we need to make sure that a Boundary Event copied and pasted on canvas is converted to an Intermediate Event.
This might be helpful: 
* [CreateBoundaryEventBehavior](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/modeling/behavior/CreateBoundaryEventBehavior.js)
* [DetachEventBehavior](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/modeling/behavior/DetachEventBehavior.js)

File: test/spec/features/copy-paste/BpmnCopyPasteSpec.js (hunk 3 above)
Comment (oguzeroglu) [reply test/spec/features/copy-paste/BpmnCopyPasteSpec.js]: Fixed it now with @barmac comment, after the fix this test failed :)
